src/view/com/auth/create/Step3.tsx

```diff
@@ -31,11 +34,11 @@ export const Step3 = observer(function Step3Impl({
           editable
           onChange={model.setHandle}
           // TODO: Add explicit text label
-          accessibilityLabel="User handle"
+          accessibilityLabel={_(msg`User handle`)}
           accessibilityHint="Input your user handle"
         />
         <Text type="lg" style={[pal.text, s.pl5, s.pt10]}>
-          Your full handle will be{' '}
+          <Trans>Your full handle will be </Trans>
```

Comment:
This kind of thing is going to cause problems -- we don't want trailing spaces to be in the translations because they won't be reliably preserved, and then we'll get spacing issues

Reply:
Good point. Let me find all these occurrences and fix them.